Automattic/wc-calypso-bridge
# i18n: Add a mechanism to update the files in the languages/ folder

### Changes proposed in this Pull Request:



Related to https://github.com/Automattic/i18n-issues/issues/594,  https://github.com/Automattic/wc-calypso-bridge/issues/1096.

This library [loads `.mo` files in the `languages` directory](https://github.com/Automattic/wc-calypso-bridge/blob/01dfb42a44d4c52130c19eb5d1aece4cdf6bdf51/class-wc-calypso-bridge.php#L203) to show translated strings to the user. However, this folder has not been updated in many years. This leads to instances where new features are untranslated to non-English users. 

This PR adds a `makepot` entry into `package.json` which generates a `.pot` file that contains the latest strings, as well as downloading the latest translations from GlotPress. The functionality is similar the system found in [wpcomsh](https://github.com/Automattic/wpcomsh/tree/trunk#how-the-translation-system-works), and we're looking to replicate that process here.







### How to test the changes in this Pull Request:




1. Checkout this commit
2. Run `npm run makepot`
3. Verify that the `languages/` folder has `wc-calypso-bridge.pot` as well as updated `.po` and `.mo` files



### Other information:




### FOR PR REVIEWER ONLY:
